dhis-2/pom.xml

```diff
@@ -1,1277 +1,1283 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
It would be great to figure out why the entire file is reformatted, this makes it hard to see what _really_ changed and will probably be changed back in a later commit.